src/bin/commands/images.rs

```diff
@@ -125,19 +143,27 @@ fn get_cross_images(
     local: bool,
 ) -> cross::Result<Vec<Image>> {
     let stdout = cross::docker::subcommand(engine, "images")
-        .arg("--format")
-        .arg("{{.Repository}}:{{.Tag}} {{.ID}}")
+        .args(&["--format", "{{.Repository}}:{{.Tag}} {{.ID}}"])
         .run_and_get_stdout(verbose)?;
 
-    let mut images: Vec<Image> = stdout
+    let mut images: BTreeSet<Image> = stdout
         .lines()
         .map(parse_image)
         .filter(|image| is_cross_image(&image.repository))
         .filter(|image| local || !is_local_image(&image.tag))
         .collect();
-    images.sort();
 
-    Ok(images)
+    let stdout = cross::docker::subcommand(engine, "images")
+        .args(&["--format", r#"{{.Repository}}:{{.Tag}} {{.ID}}"#])
```

Comment:
I don't think this needs an escape sequence, does it?

Reply:
oh yeah, that's a remenant of me trying to hack the go formatter to get the labels

sadly its not feasible, you can get them, but I don't know how well supported that is

`{{json .}}` i think was how I got them